Ginger/Ginger/App.xaml.cs

```diff
@@ -242,6 +244,13 @@ private void Application_Startup(object sender, StartupEventArgs e)
         {            
             Amdocs.Ginger.CoreNET.log4netLib.GingerLog.InitLog4Net();
 
+            if (System.Diagnostics.Trace.Listeners.Contains(CustomTraceListener.Instance) == false)
```

Comment:
Use Ginger reporter log for logging

Reply:
this is for alm repository listener is up or not